modules/OCPPConfiguration/OCPPConfiguration.cpp

```diff
@@ -0,0 +1,49 @@
+// SPDX-License-Identifier: Apache-2.0
+// Copyright Pionix GmbH and Contributors to EVerest
+#include "OCPPConfiguration.hpp"
+#include "main/event_handler.hpp"
+#include <everest/logging.hpp>
+
+namespace module {
+
+void OCPPConfiguration::init() {
+    invoke_init(*p_main);
```

Comment:
You should verify the configuration parameters in the init:
* Check if mapping file exists and is valid and warn if it doesn't
* Check if user config exists and warn if it doesn't

In case the configuration could not be verified this module is not operational, but it should not crash, since it doesn't provide any interface and shold not cause a crash of everest ever.

Reply:
The first case is handled implicitly now but I think the second case shouldn't generate a warning and just silently create the file. Do you disagree?